tooling-tests/src/test/resources/test-src/reference/tool_test_init_5/Config.toml

```diff
@@ -0,0 +1,8 @@
+oo1 = "bar1"
```

Comment:
`tool_test_init_4` isn't there. Let's rename `tool_test_init_5` -> `tool_test_init_4` and the subsequent test sources.

Reply:
tool_test_init_4 is uses to test the outside ballerina project test case